exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Content.java

```diff
@@ -30,36 +31,20 @@
     private static final String URL = "url";
     private static final String VARIANTS = "variants";
     private static final String VARIANT = "variant=";
-
-
+    public static final String PARENT = "parent";
     private Utils.CONTENT_TYPES subtype = null;
-    private List<Content> children = null;
+
 
     public Content(Utils.CONTENT_TYPES subtype){
         this.subtype = subtype;
     }
 
-    public ExpObservable<List<Content>> getChildren() {
-        if (this.children == null) {
-            final String uuid = getString(Utils.UUID);
-            final ExpObservable<Content> observable = Exp.getContent(uuid);
-            return new ExpObservable<List<Content>>(observable.<List<Content>>flatMap(new Func1<Content, Observable<List<Content>>>() {
-                @Override
-                public Observable<List<Content>> call(Content content) {
-                    Content.this.children = content.children;
-                    return Observable.just(content.children);
-                }
-            }));
-        }
-
-        return new ExpObservable<List<Content>>(Observable.just(this.children));
-    }
-
-    public void setChildren(List<Content> children){
-        this.children = children;
+    public ExpObservable<SearchResults<Content>> getChildren(Map options) {
```

Comment:
Can we make SearchResults extend List that would allow this to be a non breaking change. 

Also it would be good if we can have both getChildren() and getChildren(Map<String, Object> options)